src/simplay-web/src/StoreSetContentEventArgs.ts

```diff
@@ -0,0 +1,17 @@
+import { EventArgs } from './EventArgs';
+
+export class StoreSetContentEventArgs extends EventArgs {
+  readonly content: {
```

Comment:
Can you create a type for that in the same class? I think that will be better readable